.github/actions/unit-tests/action.yml

```diff
@@ -0,0 +1,68 @@
+name: Terraform Unit Tests
+description: Runs Terraform Unit Tests Via TerraTest
+inputs:
+  GITHUB_TOKEN:
+    description: Workflow GitHub Token (limited permissions)
+    required: true
```

Comment:
🔧  Same as previous, we shouldn't need to use an input here.

Reply:
removed!